spacy/cli/debug_data.py

```diff
@@ -954,6 +1005,20 @@ def _get_labels_from_spancat(nlp: Language) -> Dict[str, Set[str]]:
     return labels
 
 
+def _get_labels_from_trainable_lemmatizer(nlp: Language) -> Set[str]:
+    pipe_names = [
+        pipe_name
+        for pipe_name in nlp.pipe_names
+        if nlp.get_pipe_meta(pipe_name).factory == "trainable_lemmatizer"
+    ]
+    labels: Set[str] = set()
+    for pipe_name in pipe_names:
+        pipe = nlp.get_pipe(pipe_name)
+        for tree_id in range(len(pipe.trees)):
+            labels.add(pipe.trees.tree_to_str(tree_id))
```

Comment:
`n=510`

(this should also maybe not be called `labels` to avoid confusion since these arent the actual labels, but we'll get to that later`